test/commands/commerce/store/quickstart/setup.test.ts

```diff
@@ -91,6 +92,95 @@ const testFile = `<?xml version="1.0" encoding="UTF-8"?>
 </Network>
 `;
 
+describe('commerce:store:qickstart:setup', () => {
+    const config = stubInterface<IConfig>($$.SANDBOX, {});
+    const logger = new Logger('test');
+    let loggerStub: sinon.SinonStub;
+    let forceDataSoqlStub: sinon.SinonStub;
+    let forceDataRecordCreateStub: sinon.SinonStub;
+    let getApplicationContextFromWebStoreIdStub: sinon.SinonStub;
+    let getCurrencySettingsStub: sinon.SinonStub;
+    let queryResult: Result<QueryResult>;
+    let storeQuickstartSetup: StoreQuickstartSetup;
+
+    before(() => {
+        queryResult = new Result<QueryResult>();
+        queryResult.result = new (class implements QueryResult {
+            public done: boolean;
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            public records: Record[] = [{ Id: 'testId' }];
+            public totalSize: number;
+        })();
+        storeQuickstartSetup = new StoreQuickstartSetup([], config);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any,@typescript-eslint/no-unsafe-member-access
+        (storeQuickstartSetup as any).varargs = { communitySiteName: 'test', getUsername: 'test' };
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        storeQuickstartSetup.ux = stubInterface<UX>($$.SANDBOX);
+    });
+
+    after(() => {
+        sinon.restore();
+    });
+
+    beforeEach(() => {
+        loggerStub = sinon.stub(logger, 'debug').returns(logger);
+        forceDataSoqlStub = stub(forceOrgSoqlExports, 'forceDataSoql');
+        forceDataSoqlStub.returns(queryResult);
+        forceDataRecordCreateStub = stub(forceOrgSoqlExports, 'forceDataRecordCreate');
+        forceDataRecordCreateStub.returns(null);
+        getApplicationContextFromWebStoreIdStub = stub(
+            storeQuickstartSetup,
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            'getApplicationContextFromWebStoreId' as any
+        ).resolves(JSON.parse('{"country":"US", "defaultCurrency":"USD"}'));
+    });
+
+    afterEach(() => {
+        loggerStub.restore();
+        forceDataSoqlStub.restore();
+        forceDataRecordCreateStub.restore();
+        getApplicationContextFromWebStoreIdStub.restore();
+    });
+
+    it('shouldPopulateCurrencyIsoCodeIfOrgIsMulticurrency', async () => {
```

Comment:
You can write in a human readable format here, no need for camelCase. Spaces are fine.